recipes/libsafec/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibSafeCConan(ConanFile):
+    name = "libsafec"
+    description = "This library implements the secure C11 Annex K[^5] functions" \
+            " on top of most libc implementations, which are missing from them."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rurban/safeclib"
+    license = "MIT"
+    topics = ("conan", "safec", "libc")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    build_requires = "autoconf/2.69", "libtool/2.4.6"
+    exports_sources = "patches/*"
+
+    __autotools = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _supported_compiler(self):
+        compiler = self.settings.compiler
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == "Visual Studio":
+            return False
+        if compiler == "gcc" and version < "5":
+            return False
+        return True
+
+    def configure(self):
+        if not self._supported_compiler:
+            raise ConanInvalidConfiguration(
+                "libsafec doesn't support compiler: {} on OS: {}.".format(
+                    self.settings.compiler, self.settings.os))
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("safeclib-*")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    @property
+    def _autotools(self):
+        if self.__autotools:
+            return self.__autotools
+        self.run("autoreconf -fiv", run_environment=True)
+        self.__autotools = AutoToolsBuildEnvironment(self)
+        if self.options.shared:
+            args = ["--enable-shared", "--disable-static"]
+        else:
+            args = ["--disable-shared", "--enable-static"]
+        args.extend(["--disable-doc", "--disable-dependency-tracking"])
+        if self.settings.build_type in ("Debug", "RelWithDebInfo"):
+            args.append("--enable-debug")
+        self.__autotools.configure(args=args)
+        return self.__autotools
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Makefile.am"), "_@SAFEC_API_VERSION@", "")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Makefile.am"), "-@SAFEC_API_VERSION@", "")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "tests", "Makefile.am"), "-@SAFEC_API_VERSION@", "")
```

Comment:
This looks incorrect 🤔 

https://github.com/rurban/safeclib/blob/7ef68891f130e257bf7e442b0dce626df7dcf068/configure.ac#L1008 sets it to the version

and it affects the output of the libs

Reply:
Done: https://github.com/conan-io/conan-center-index/pull/4434/commits/d68956cbd9925904ced61446fce68a4c01e60ce0